yts/yts.py

```diff
@@ -124,146 +136,100 @@ def extract_movie_data(self, movie_data) -> dict:
         )
         return {movie_title: rating}
 
+
     # Execute Transmission-gtk with the downloaded torrent
-    def execute_transmission(self, torrent_name):
-        command = f"transmission-gtk {torrent_name}"
-        command_list = command.split()
-        subprocess.Popen(command_list)
+    @staticmethod
+    def execute_transmission(torrent_name):
+        subprocess.Popen(["transmission-gtk", torrent_name])
 
     # Utitility method to print available formats
     # Param: return of extract_format method
-    def print_formats(self, formats):
+    @staticmethod
```

Comment:
Remove the `@staticmethod` and add `self` for now as print_formats uses get_movie_data instance method.
I'll redesign the YTS class with static methods. : )